.ci/azure/linux.yml

```diff
@@ -168,6 +147,36 @@ jobs:
     workingDirectory: $(WORK_DIR)
     displayName: 'Install dependencies'
 
+  - script: |
+      wget "https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}.tar.gz"
```

Comment:
maybe we need to update minimum version in install_build_dependencies.sh ? And better to use prebuild cmake in install_build_dependencies.sh instead of compiling our one

Reply:
I won't change this script because it install dependencies not only for x86_64 platform.
For CI I use prebuild cmake version